lld/test/ELF/loongarch-call36-shared.s

```diff
@@ -0,0 +1,41 @@
+# REQUIRES: loongarch
```

Comment:
We can use one test to test non-PLT (executable link) and PLT (`-shared`).

Reply:
Done.